composer.json

```diff
@@ -21,7 +21,6 @@
     "pimcore/google-marketing-bundle": "^1.0",
     "knplabs/knp-paginator-bundle": "^6.0.0",
```

Comment:
Do we need to change the pimcore version too?

Reply:
Same topic as https://github.com/pimcore/customer-data-framework/pull/540#discussion_r1735934166